src/bin/pg_autoctl/parsing.c

```diff
@@ -220,7 +223,9 @@ bool
 parse_controldata(PostgresControlData *pgControlData,
 				  const char *control_data_string)
 {
-	if (!parse_controldata_field_uint32(control_data_string,
+	if (!parse_controldata_field_dbstate(control_data_string,
+										 &(pgControlData->state)) ||
```

Comment:
Extra newline to be consistent with the rest of this function
```suggestion
										 &(pgControlData->state)) ||
										 
```

Reply:
Fixed.